.tekton/collector-pull-request.yaml

```diff
@@ -221,12 +225,27 @@ spec:
       - name: basic-auth
         workspace: git-auth
 
+    - name: prepare-rhel-rpm-subscriptions
+      runAfter:
+      - clone-repository
+      workspaces:
+      - name: source
+        workspace: workspace
+      - name: subscription-manager-activation-key
+        workspace: subscription-manager-activation-key
+      taskSpec:
+        steps:
+        # TODO(ROX-20651): use content sets instead of subscription manager for access to RHEL RPMs once available.
+        - name: smuggle-activation-key
+          image: registry.access.redhat.com/ubi8/ubi:latest
+          script: exec /workspace/source/source/.konflux/scripts/subscription-manager-bro.sh smuggle
+
     - name: prefetch-dependencies
       params:
       - name: input
         value: $(params.prefetch-input)
       runAfter:
-      - clone-repository
+      - prepare-rhel-rpm-subscriptions
```

Comment:
Now, after seeing how collector-full pipeline graph is arranged, I'd nitpick on this.

You have:

```yaml
    - name: prepare-rhel-rpm-subscriptions
      runAfter:
      - clone-repository
```
I agree with this ^ one.

```yaml
    - name: prefetch-dependencies
      runAfter:
      - prepare-rhel-rpm-subscriptions
```
Prefetching dependencies does not need to depend on `prepare-rhel-rpm-subscriptions`. It can remain depending on `clone-repository`.

```yaml
    - name: fetch-support-package
      runAfter:
      - init
      - clone-repository
```
`clone-repository` already depends on `init`, therefore `fetch-support-package` does not need `init` in its list.

```yaml
    - name: build-container
      runAfter:
      - prefetch-dependencies
      - fetch-support-package
```
This would need `prepare-rhel-rpm-subscriptions` additionally in its list.

Basically, either all three `prepare-rhel-rpm-subscriptions`, `prefetch-dependencies` and `fetch-support-package` need to run in parallel with each other or sequentially for consistency. Currently, it's a mix of both and it's unclear whether that's on purpose.

Reply:
https://github.com/stackrox/collector/pull/1573/commits/c41368a927d845cdf71235ecaef27abda93bef80

Reply:
![Screenshot 2024-02-28 at 14 50 49](https://github.com/stackrox/collector/assets/9576848/dc82f559-27ef-488e-86b0-16dcbd5f4897)

This looks better.